package.json

```diff
@@ -19,8 +19,10 @@
   "homepage": "https://github.com/bloglovin/lintlovin",
   "dependencies": {
     "dependency-check": "^2.1.0",
+    "grunt": "^0.4.5",
```

Comment:
I was about to say that I didn't add it, but the diff says I did. I think it was automatically added by the npm install --save command... *sigh*